ddl/src/test/java/com/adaptivescale/rosetta/ddl/test/SQLServerDDLTest.java

```diff
@@ -24,25 +24,44 @@ public void createDB() throws IOException {
                 "SELECT  *    FROM sys.schemas    WHERE name = N'test')" +
                 "EXEC('CREATE SCHEMA test');" +
                 "CREATE TABLE \"test\".\"Categories\"(\"CategoryID\" int identity NOT NULL , \"CategoryName\" nvarchar NOT NULL , \"Description\" ntext NOT NULL , \"Picture\" image NOT NULL , PRIMARY KEY (\"CategoryID\"));" +
+                "IF NOT EXISTS (SELECT  *    FROM sys.schemas    WHERE name = N'test')EXEC('CREATE SCHEMA test');\n" +
```

Comment:
as can be seen here - we have repetitve create shema - we should lease the test as they were before and only add schema check/creation only at the beggining of ddl